user-service/src/test/java/rs/banka4/user_service/unit/employee/EmployeeServiceCreateTests.java

```diff
@@ -27,12 +37,35 @@ public class EmployeeServiceCreateTests {
     private EmployeeMapper employeeMapper;
     @Mock
     private UserService userService;
+    @Mock
+    private JwtService jwtService;
     @InjectMocks
     private EmployeeServiceImpl employeeService;
 
     @BeforeEach
     void setUp() {
         MockitoAnnotations.openMocks(this);
+
+        // Mock the security context
+        SecurityContext securityContext = mock(SecurityContext.class);
+        Authentication authentication = mock(AuthenticatedBankUserAuthentication.class);
+        AuthenticatedBankUserPrincipal principal = mock(AuthenticatedBankUserPrincipal.class);
```

Comment:
please don't mock data classes - these can be easily constructed with the needed information (they're written with that in mind)